src/bci_build/package/__init__.py

```diff
@@ -261,11 +261,9 @@ def release_package_names(self) -> tuple[str, ...]:
             return ("SLES-release",)
         if self.is_ltss:
             return ("sles-ltss-release",)
-        # if self.is_slcc:
-        #     return (f"{str(self.value).lower()}-release",)
 
         assert self.is_sle15
-        return ("sles-release",)
+        return ("sles-release", "coreutils")
```

Comment:
The `coreutils` package is added to the busybox container too, where it causes conflicts

Reply:
ok, then we do this only for the fips containers, all the others anyway already had coreutils in the list